mappings/net/minecraft/entity/ai/pathing/PathNodeMaker.mapping

```diff
@@ -0,0 +1,53 @@
+CLASS none/bab net/minecraft/entity/ai/pathing/PathNodeMaker
+	FIELD a blockView Lnone/aja;
+	FIELD b entity Lnone/sq;
+	FIELD c pathNodeCache Lnone/om;
+	FIELD d xSize I
+	FIELD e ySize I
+	FIELD f zSize I
+	FIELD g canEnterOpenDoors Z
+	FIELD h canPathThroughDoors Z
+	FIELD i canSwim Z
```

Comment:
These shouldn't be prefixed with `can` if possible, that's reserved for methods,